pynq/lib/_pynq/_audio/audio_adau1761.cpp

```diff
@@ -328,9 +333,11 @@ extern "C" void bypass(unsigned int audio_mmap_size,
     // Enable Mixer3 and Mixer4
     write_audio_reg(R22_PLAYBACK_MIXER_LEFT_CONTROL_0, 0x21, iic_fd);
     write_audio_reg(R24_PLAYBACK_MIXER_RIGHT_CONTROL_0, 0x41, iic_fd);
+    
+    unsigned char vol_register = (unsigned char)volume << 2 | 0x3;
     // Enable Left/Right Headphone out
-    write_audio_reg(R29_PLAYBACK_HEADPHONE_LEFT_VOLUME_CONTROL, 0xE7, iic_fd);
-    write_audio_reg(R30_PLAYBACK_HEADPHONE_RIGHT_VOLUME_CONTROL, 0xE7, iic_fd);
+    write_audio_reg(R29_PLAYBACK_HEADPHONE_LEFT_VOLUME_CONTROL, vol_register, iic_fd);
```

Comment:
Make sure lines do not exceed 79 characters per line. Break long line if needed.

Reply:
Thanks for the nice review.